object/user.go

```diff
@@ -114,6 +114,8 @@ type User struct {
 
 	LastSigninWrongTime string `xorm:"varchar(100)" json:"lastSigninWrongTime"`
 	SigninWrongTimes    int    `json:"signinWrongTimes"`
+
+	ManagedAccounts []ManagedAccount `xorm:"managedAccounts blob" json:"managedAccounts"`
```

Comment:
why not use `varchar(xxx)`?

Reply:
I use the same struct as WebauthnCredentials `WebauthnCredentials []webauthn.Credential `xorm:"webauthnCredentials blob" json:"webauthnCredentials"``

Reply:
OK